src/contracts/contract_deployer.star

```diff
@@ -217,13 +188,21 @@ def deploy_contracts(plan, priv_key, l1_config_env_vars, optimism_args, l1_netwo
         ],
         files={
             "/network-data": op_deployer_init.files_artifacts[0],
+            "/tmp": intent_json_artifact,
         },
         run=" && ".join(
             [
-                "dasel put -r toml -t {0} -v {2} '{1}' -o /network-data/intent.toml < /network-data/intent.toml".format(
-                    t, k, v
-                )
-                for t, k, v in intent_updates
+                # zhwrd: this mess is temporary until we implement json reading for op-deployer intent file
```

Comment:
😱 oh my

Do we have a ticket to track the op-deployer improvement that'll allow deleting this? :)

Reply:
i'm happy to build this right away if we need it